code/api/src/modules/crate/mutations.test.js

```diff
@@ -0,0 +1,87 @@
+import request from 'supertest'
+import express from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+import authentication from '../../setup/authentication'
+import models from '../../setup/models'
+import { isType } from 'graphql'
+
+
+
+describe('crate mutations', () => {
+  let server;
+  let token;
+  let crateId;
+
+  beforeAll(async () => { // get the server running and attached for each of our future tests
+    server = express();
+
+    server.use(authentication);
+
+    server.use(
+      '/',
+      graphqlHTTP(request => ({
+        schema: schema, // import the schema
+        graphiql: true,
+        context: {
```

Comment:
need to bring `auth` into the graphqlHTTP. This came directly from api/src/setup/graphql.js